packages/j/jellyfin-media-player/package.yml

```diff
@@ -0,0 +1,27 @@
+name       : jellyfin-media-player
+version    : 1.11.1
+release    : 1
+source     :
+    - https://github.com/jellyfin/jellyfin-media-player/archive/refs/tags/v1.11.1.tar.gz : 75499ed2721b77ea0f757da20615aff8e5e9d8e9ff9d4b2572e71067be17ea29
+homepage   : https://jellyfin.org/
+license    : GPL-2.0-or-later
+component  : multimedia.video
+summary    : Jellyfin Desktop Client
+description: |
+    Desktop client using jellyfin-web with embedded MPV player. Supports Windows, Mac OS, and Linux. Media plays within the same window using the jellyfin-web interface unlike Jellyfin Desktop. Supports audio passthrough.
+builddeps  :
+    - pkgconfig(Qt5Core)
+    - pkgconfig(Qt5WebChannel)
+    - pkgconfig(Qt5WebEngineWidgets)
+    - pkgconfig(Qt5X11Extras)
+    - pkgconfig(libcec)
+    - pkgconfig(mpv)
+    - pkgconfig(sdl2)
+rundeps    :
+    - qt5-quickcontrols
```

Comment:
Generally I've been switching Qt applications to use Clang by default since that's the toolchain I use to build Qt. Please add the following:
```
clang : yes
optimize : thin-lto
```